gno.land/pkg/sdk/vm/keeper.go

```diff
@@ -73,20 +77,36 @@ func NewVMKeeper(
 	return vmk
 }
 
-func (vm *VMKeeper) Initialize(ms store.MultiStore) {
+func (vm *VMKeeper) Initialize(logger *slog.Logger, ms store.MultiStore) {
 	if vm.gnoStore != nil {
 		panic("should not happen")
 	}
 	alloc := gno.NewAllocator(maxAllocTx)
 	baseSDKStore := ms.GetStore(vm.baseKey)
 	iavlSDKStore := ms.GetStore(vm.iavlKey)
 	vm.gnoStore = gno.NewStore(alloc, baseSDKStore, iavlSDKStore)
-	vm.gnoStore.SetPackageGetter(vm.getPackage)
 	vm.gnoStore.SetNativeStore(stdlibs.NativeStore)
-	if vm.gnoStore.NumMemPackages() > 0 {
+	if vm.gnoStore.NumMemPackages() == 0 {
+		// No packages in the store; set up the stdlibs.
+		start := time.Now()
+
+		stdlibInitList := stdlibs.InitOrder()
+		for _, lib := range stdlibInitList {
+			if lib == "testing" {
```

Comment:
I think that testing makes sense to be completely skipped in the context of gno.land.

Reply:
I used to agree; but now I think it makes sense, so we can build things like assert, or testing helper libraries, and have them published (and type checked) on-chain.

In #2425, I'm trying to join up the normal stdlibs and the testing stdlibs for this very eeason.